nodePackages.jshint fails because it cannot connect to github.com

## Issue description

Reproducing is pretty simple: `nix-shell -p nodePackages.jshint`.

The error is:

```
> phantomjs-prebuilt@2.1.16 install /nix/store/in2lf997vccm7pjpi0cjg3ian8cwd5i5-node-jshint-2.9.6/lib/node_modules/jshint/node_modules/phantomjs-prebuilt
> node install.js

PhantomJS not found on PATH
Downloading https://github.com/Medium/phantomjs/releases/download/v2.1.1/phantomjs-2.1.1-linux-x86_64.tar.bz2
Saving to /build/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
Receiving...

Error making request.
Error: getaddrinfo ENOTFOUND github.com github.com:443
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)

Please report this full log at https://github.com/Medium/phantomjs
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! phantomjs-prebuilt@2.1.16 install: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the phantomjs-prebuilt@2.1.16 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.18.7, NixOS, 19.03pre153158.7df10f388da (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.2`
 - channels(philipp): `"nixpkgs-19.03pre151875.7c1b85cf6de"`
 - channels(root): `"nixos-19.03pre153158.7df10f388da"`
 - nixpkgs: `/home/philipp/.nix-defexpr/channels/nixpkgs`

--- a/pkgs/development/node-packages/default-v8.nix
+++ b/pkgs/development/node-packages/default-v8.nix
@@ -16,6 +16,10 @@ nodePackages // {
     '';
   };
 
+  jshint = nodePackages.jshint.override {
+    buildInputs = [ pkgs.phantomjs2 ];
+  };
+
   dat = nodePackages.dat.override {
     buildInputs = [ nodePackages.node-gyp-build ];
   };

